### ISSUE TITLE
MOS(test) Hifi-Gan traditional vs yours

### ISSUE THREAD

**Issue by skol101** [yuan1615/AdaVocoder#5]
Hello,

What would you attribute the difference in MOS(test) scores of yours vs traditional, when cross domain consistence loss is introduced only at  AdaHiFi-GAN train step? 
<img width="596" alt="Screenshot 2022-12-27 at 23 28 54" src="https://user-images.githubusercontent.com/53978091/209724088-8f99079d-8e7b-4c98-8384-8574b1c81d52.png">

**Comment by yuan1615** [comment]
The traditional method is prone to overfitting, especially in the case of few-shot. You can see that while similar scores were achieved in the training set, traditional methods performed poorly in the test set.

**Comment by skol101** [comment]
Cheers!
